### ISSUE TITLE
about batch size

### ISSUE THREAD

**Issue by liuqinying** [ZhaofanQiu/pseudo-3d-residual-networks#24]
@ZhaofanQiu Hi,I am a ustcer too.
In your paper, it is said you trained P3D with multiple GPUs in parallel. And we know P3D resnet has batch norm layers.So I wonder whether the bacth norm layers use the data in each gpu or all gpus  (to calculate mean, variance , etc.)?
Any help will be greatly appreciated
